staging/src/slime.io/slime/modules/meshregistry/pkg/source/zookeeper/sidecar.go

```diff
@@ -187,6 +220,67 @@ func (s *Source) refreshSidecar(init bool) {
 	}
 }
 
+func (s *Source) refreshSidecarMockServiceEntry() {
+	se := &networking.ServiceEntry{
+		Hosts:      []string{mockService},
+		Ports:      make([]*networking.Port, 0),
+		Resolution: networking.ServiceEntry_STATIC,
+	}
+
+	s.mut.RLock()
```

Comment:
Will all tasks sharing the same lock affect each other's operating efficiency？

Reply:
no. the `locked`(especially `rlocked`) code block is very small